TangramMap.framework/Headers/Helpers.h

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
Yep it does, I've just noticed an error in our cmake config https://github.com/tangrams/tangram-es/blob/master/toolchains/iOS.framework.cmake#L50. It's not harmful for now unless you include this file, it will go away on next update of the framework.